releasenotes/notes/remove-tracer-deprecations-c0abaecda9c24b79.yaml

```diff
@@ -0,0 +1,17 @@
+---
+upgrade:
+  - |
+    Remove deprecated attribute ``ddtrace.Tracer.debug_logging``. Set the logging level for the ``ddtrace`` logger instead::
```

Comment:
Don't think this is accurate since `tracer.log` is the `ddtrace.tracer.log` logger not the root `ddtrace` logger.